### PR TITLE
Set key binding to expand all aliases including global

### PR DESCRIPTION
**Pull request by akarzim**
control-space expands all aliases, including global.

### CODE REVIEW

File: modules/editor/init.zsh [hunk 1]
```diff
@@ -368,6 +376,9 @@ for keymap in 'emacs' 'viins'; do
 
   # Insert 'sudo ' at the beginning of the line.
   bindkey -M "$keymap" "$key_info[Control]X$key_info[Control]S" prepend-sudo
+
+  # control-space expands all aliases, including global
+  bindkey -M "$keymap" "$key_info[Control] " globalias
```
Comment (belak) [inline modules/editor/init.zsh]: Should this be glob-alias, similar to above?

Comment (akarzim) [reply modules/editor/init.zsh]: My bad. I fixed it.

**Comment by akarzim** [comment]
Which built in expansion?

The reason behind `^<Space>` is to expand on demand, not each time I press the space bar.

**Comment by belak** [comment]
I'm still not exactly sure what this does, but I haven't had much time to look at it yet either. If you could give an example of where you use this, that would help me understand what this does.

**Comment by akarzim** [comment]
Here is an example.

```sh
alias rbb=bundle
alias rbbi='bundle install --path vendor/bundle'
alias -g G='| grep'
```

globalias zsh

**Pull request merged**